src/udev/udev-rules.c

```diff
@@ -254,6 +257,48 @@ struct UdevRules {
                               "Invalid value \"%s\" for %s (char %zu: %s), ignoring.", \
                               value, key, offset, hint)
 
+#define log_line_full_errno_zerook(rule_file, line_nr, level, error, fmt, ...) \
```

Comment:
Makes sense.

Reply:
Yes, it would be nice to have just one argument instead of two, but there is one place in the code where `Line is too long` error is logged but there is no associated `UdevRuleLine` object because `rule_add_line` has not been called for that invalid line.

What would you suggest?

Reply:
Indeed. Let's cleanup the macros later, if we find a nicer way.